changelog.txt

```diff
@@ -7,6 +7,7 @@
 * Fix - WooCommerce Payments disappeared from WooCommerce Settings if WooCommerce Subscriptions is activated.
 * Add - REST endpoint to capture payments by order ID
 * Add - Payment request button support for checkout and cart blocks.
+* Fix - Risk level is displayed as a "Numeric" value in transactions CSV #1720'.
```

Comment:
I just noticed there is a closing single quote here. Is that intentional?

Reply:
Aah! my bad, removed it!